bucket/falkon.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://www.falkon.org",
+    "version": "3.1.0",
+    "description": "KDE web browser using QtWebEngine rendering engine, previously known as QupZilla.",
+    "license": "GPL-3.0-or-later",
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.kde.org/stable/falkon/3.1/Falkon.Installer.3.1.0.x64.exe#/dl.7z",
+            "hash": "5ab4c139dee19c21114f7acbe38c40babb392bfb875cb977db11fd390b7aa7ee"
+        },
+        "32bit": {
+            "url": "https://download.kde.org/stable/falkon/3.1/Falkon.Installer.3.1.0.exe#/dl.7z",
+            "hash": "806dc30fd68edee4587e2f480054000f82d5d56df5047ee299547cb2247a3a88"
+        }
+    },
+    "bin": "falkon.exe",
+    "shortcuts": [
+        [
+            "falkon.exe",
+            "Falkon"
+        ]
+    ],
+    "checkver":
+    "checkver": "Current version: ([\\d.]+)",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://download.kde.org/stable/falkon/$majorVersion.$minorVersion/Falkon.Installer.$version.x64.exe#/dl.7z"
+            },
+            "32bit": {
+                "url": "https://download.kde.org/stable/falkon/$majorVersion.$minorVersion/Falkon.Installer.$version.exe#/dl.7z"
+            },
+            "hash": {
```

Comment:
Remove this property. You moved it into wrong scope

Reply:
done